sagemaker-fundamentals/create-training-job/container_kms/local_test/ml/input/config/hyperparameters.json

```diff
@@ -0,0 +1,8 @@
+{
+    "ciphertext_s3_key": "dont_look",
+    "master_key_id": "b54b4c32-e8bf-4918-b030-b3aeff5b346d",
```

Comment:
Don't use something that looks like a real AWS KMS key.
use `1234abcd-12ab-34cd-56ef-1234567890ab`

Reply:
That's right, I disabled it.